alf/algorithms/mcts_algorithm.py

```diff
@@ -505,19 +646,59 @@ def _search(self, trees, to_plays):
         children_index = trees.children_index
         best_child_index = trees.best_child_index
         game_over = trees.game_over
+        psims = self._num_parallel_sims
+        parallel = self._parallel
         search_paths = []
         B = trees.B
         nodes = trees.root_indices
-        path_lengths = torch.ones_like(B)
+        calc_i = False
```

Comment:
I think it is worth explaining what `i` is here, since this part of code is already hundreds lines away from where "parallel sims" is explained. Or give a pointer to where "parallel sims" is explained above (or below).